planemo/galaxy/config.py

```diff
@@ -1240,6 +1254,7 @@ def _install_with_command(ctx, config_directory, command, env, kwds):
     else:
         pip_install_command = ""
     setup_venv_command = setup_venv(ctx, kwds)
+    env['__PYVENV_LAUNCHER__'] = ''
```

Comment:
I guess the cleaner fix would be to omit `__PYVENV_LAUNCHER__` in galaxy-lib, that'd also be more in line with https://github.com/python/cpython/pull/9516. What do you think @nsoranzo ?

Reply:
Where exactly in galaxy-lib ? And at which of the joined commands below `__PYVENV_LAUNCHER__` starts creating issues?

Reply:
https://github.com/galaxyproject/galaxy/blob/dev/lib/galaxy/tools/deps/commands.py#L88

It's hard to tease apart exactly where in the joined command that happens, but the issue is that shebang for all scripts in the newly created virtualenv points to planemo's virtualenv instead of the newly created virtualenv.